src/module/actor/vehicle/data.ts

```diff
@@ -52,17 +53,10 @@ interface VehicleSystemData extends ActorSystemData {
         speed: number;
     };
     saves: {
-        fortitude: {
-            rank: number;
-            value: number;
-            saveDetail: string;
-        };
+        fortitude: StatisticModifier;
```

Comment:
This should probably be SaveData instead. Or something equivalent that extends StatisticCompatData.